src/app/reference/controllers/benthic-attributes.ctrl.js

```diff
@@ -63,27 +64,64 @@ angular.module('app.reference').controller('BenthicAttributesCtrl', [
             return regions.join(', ') || '-';
           }
         }
-      ]
+      ],
+      toolbar: {
+        template: 'app/reference/partials/fish-families-toolbar.tpl.html',
+        clearFilters: function() {
+          $scope.tableControl.clearSearch();
+        }
+      }
     };
 
-    var promise = offlineservice.BenthicAttributesTable();
+    const updateBenthicAttributeCount = function() {
+      $scope.projectObjectsTable.count().then(function(count) {
+        benthicAttributeRecordsCount = count;
+      });
+    };
+
+    const promise = offlineservice.BenthicAttributesTable();
     promise.then(function(table) {
+      $scope.projectObjectsTable = table;
+      updateBenthicAttributeCount();
       $scope.resource = new PaginatedOfflineTableWrapper(table, {
         searchFields: ['name', '$$benthicattributes.name']
       });
-      table.filter().then(function(records) {
-        var joinSchema = {
+      $scope.projectObjectsTable.filter().then(function(records) {
+        const joinSchema = {
           benthicattributes: {
             foreignKey: 'parent',
             relatedRecords: records,
             relatedKey: 'id',
             relatedColumns: ['name']
           }
         };
-        table.setJoinDefn(joinSchema);
+        $scope.projectObjectsTable.setJoinDefn(joinSchema);
       });
+      $scope.projectObjectsTable.$watch(
+        updateBenthicAttributeCount,
+        null,
+        'benthicAttributeRecordsCount'
+      );
     });
 
+    $scope.tableControl.getFilteredRecordsCount = function() {
+      const tableRecordsTotal =
+        $scope.tableControl.getPaginationTable() &&
+        $scope.tableControl.getPaginationTable().total;
+
+      return `${tableRecordsTotal}/${benthicAttributeRecordsCount}`;
```

Comment:
We should protect against dividing by 0 here

Reply:
it's inside string literals, not dividing by 0